client/public/index.html

```diff
@@ -2,12 +2,13 @@
 <html>
 <head>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.6/flatly/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="./datetime.css">
   <link rel="stylesheet" type="text/css" href="./styles.css">
   <title>EtherList</title>
 </head>
 <body>
-<div id="app">
-</div>
- <script type="text/javascript" src="bundle.js" charset="utf-8"></script>
- </body>
+  <div id="app"></div>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/core.js"></script>
```

Comment:
Are we adding jQuery as a dep. only for $.ajax? We should use the HTML5 Fetch API instead, if that is the case